pkg/controller/kataconfig/openshift_reconciler.go

```diff
@@ -405,22 +412,61 @@ func (r *ReconcileKataConfigOpenShift) listKataPods() error {
 	return nil
 }
 
+func kataOcExists(r *ReconcileKataConfigOpenShift) (bool, error) {
```

Comment:
if you change the method signature to `func ((r *ReconcileKataConfigOpenShift) kataOcExists()` you don't have to pass a pointer to the object as an argument.

Reply:
good point :-) I'll do that, thx